gr-qtgui/lib/timeRasterGlobalData.cc

```diff
@@ -46,6 +46,13 @@ TimeRasterData::TimeRasterData(const double rows, const double cols)
 
 TimeRasterData::~TimeRasterData() {}
 
+void TimeRasterData::setInterval(Qt::Axis axis, const QwtInterval& interval)
+{
+    d_intervals[axis] = interval;
+}
+
+QwtInterval TimeRasterData::interval(Qt::Axis a) const { return d_intervals[a]; }
```

Comment:
```suggestion
#if QWT_VERSION >= 0x060200
void TimeRasterData::setInterval(Qt::Axis axis, const QwtInterval& interval)
{
    d_intervals[axis] = interval;
}

QwtInterval TimeRasterData::interval(Qt::Axis a) const { return d_intervals[a]; }
#endif
```

Reply:
Thanks for those code suggestions. I'll make those changes and test them locally first with the CI docker images and update the PR.